<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Attendance</class>
 <widget class="QMainWindow" name="Attendance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Attendance</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/images/008_Reminder_128x128_72.png</normaloff>:/images/008_Reminder_128x128_72.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="ManualClockin">
        <property name="text">
         <string>Manual Clockin/Out</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="RecalculateAllAttendance">
        <property name="text">
         <string>Recalculate Attendance</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>577</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="UploadAttendance">
        <property name="text">
         <string>Upload Attendance</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Attendance</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Past Attendance:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="dtpPast">
            <property name="date">
             <date>
              <year>2012</year>
              <month>1</month>
              <day>1</day>
             </date>
            </property>
            <property name="displayFormat">
             <string>dd-MMM-yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeView" name="lstAttendance"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Attendance Per Employee</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Employee:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="Employee">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Start Date:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="StartDate">
            <property name="displayFormat">
             <string>dd-MMM-yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>End Date:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="EndDate">
            <property name="displayFormat">
             <string>dd-MMM-yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ShowEmployeeAttendance">
            <property name="text">
             <string>Show</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="RecalculateEmployeeAttendance">
            <property name="text">
             <string>Recalculate</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeView" name="lst_EmployeeAttendance"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Attendance Report</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Start Date:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="AttStartDate">
            <property name="date">
             <date>
              <year>2012</year>
              <month>1</month>
              <day>1</day>
             </date>
            </property>
            <property name="displayFormat">
             <string>dd-MMM-yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>End Date:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="AttEndDate">
            <property name="date">
             <date>
              <year>2012</year>
              <month>1</month>
              <day>1</day>
             </date>
            </property>
            <property name="displayFormat">
             <string>dd-MMM-yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="AttShowReport">
            <property name="text">
             <string>Show</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="ReportViewer" name="AttReportWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Dail">
       <attribute name="title">
        <string>Daily Sign In Form</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Date:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="dateEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="GenerateDailySignInSheets">
            <property name="text">
             <string>Generate Form</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="ReportViewer" name="DailySignIn" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ReportViewer</class>
   <extends>QWidget</extends>
   <header>reportviewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
